modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -1039,7 +1039,7 @@
 #elif defined(NRF9120_XXAA) || defined(NRF9160_XXAA)
     #include <nrfx_config_nrf91.h>
 #else
```

Comment:
Maybe it should be done with a Kconfig so that if you really compile with unknown/unsupported device you get the error.

Reply:
> Maybe it should be done with a Kconfig so that if you really compile with unknown/unsupported device you get the error.

What kind of Kconfig do you have in mind?